types/package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "@sap-devx/app-studio-toolkit-types",
-  "publisher": "SAPOSS",
-  "version": "0.0.11",
+  "version": "1.0.0",
   "author": "SAP SE",
   "license": "Apache-2.0",
-  "description": "An npm package that includes Business Application Studio declarations",
+  "description": "The SAP Business Application Studio VSCode Ext type signatures",
```

Comment:
I think I wrote this description, but maybe we need a better one from technical write?